src/site/content/en/blog/samesite-cookie-recipes/index.md

```diff
@@ -0,0 +1,324 @@
+---
+title: SameSite cookie recipes
+subhead:
+  Update your site's cookies to prepare for the upcoming changes to the SameSite
+  attribute's behavior.
+authors:
+  - rowan_m
+date: 2019-10-15
+hero: cookie-hero.jpg
+description: |
+  With the introduction of the new SameSite=None attribute value, sites can now
+  explicitly mark their cookies for cross-site usage. Browsers are moving to
+  make cookies without a SameSite attribute act as first-party by default, a
+  safer and more privacy preserving option than the current open behavior.
+  Learn how to mark up your cookies to ensure your first-party and third-party
+  cookies continue to work once this change comes into effect.
+tags:
+  - post
+  - security
+  - cookies
+---
+
+{% Aside %}
+
+If you need a refresher on cookies and the `SameSite` attribute, head over to
+the earlier companion article,
```

Comment:
You can delete this whole line without losing meaning.

Reply:
I think what I'm looking for here is a standard widget for a series of posts. My concern is that just a link to the article doesn't explain why you would follow it. I feel like I need to provide some kind of context.

Do you think the following would work in the respective articles?

- *(In part 1)* For implementation advice on `SameSite=None`, see part 2: "`SameSite` cookie recipes"
- *(In part 2)* For how cookies and `SameSite` work, see part 1: "`SameSite` cookies explained"

Reply:
I was only suggesting deleting 'the earlier companion article'. The stuff before it is good.

Reply:
I've gone with some compact part 1  / part 2 introductions,